### DISCUSSION TITLE
Problems with the newest version 0.16.15

### DISCUSSION THREAD

**Discussion by lpetrilabsxd** [Codeneos/vlocode#364] [Q&A]
Today I update my extension to version 0.16.15 and Vlocode stop working. It wouldn't let me select any organization and the Vlocode terminal didn't show. After several attempts installing and uninstalling including Visual Studio Code. I installed version 0.16.1 and magically everything worked again

**Comment by Codeneos** [comment]
Thanks for bringing this to my attention. I usually test new versions of the extension locally before pushing them to the marketplace but this slipped through.

The cause of the issue is tsconfig-paths-webpack-plugin which affects module resolution in webpack by reading tsconfig.json for aliases. I use webpack to rollup all modules and source into a single JS file that speeds up loading times and installation time of the extension significantly.

I patched the bug tsconfig-paths-webpack-plugin, CircleCI will publish the new release (0.16.16) shortly.

**Reply by lpetrilabsxd** [reply]
You 're welcome! This extension is the best for vsc. Thanks you for do our job easy!!!
